src/core.c/Real.pm6

```diff
@@ -58,6 +58,10 @@ my role Real does Numeric {
     multi method log(Real:D: Real $base) { self.Bridge.log($base.Bridge) }
     proto method exp(|) {*}
     multi method exp(Real:D: )           { self.Bridge.exp               }
+    
+    method frac(Real:D:) {
+        self.abs - self.Int.abs
```

Comment:
Wouldn't it be a bit faster if we do `abs(self - self.Int)`? One less method call anyway.

Reply:
You are right. By my count 4 ops with my code, 3 ops with yours.